bitbots_vision/src/bitbots_vision/vision_modules/lines.py

```diff
@@ -161,6 +162,18 @@ def get_line_mask(self):
             self._white_mask = cv2.medianBlur(white_mask, 3)
         return self._white_mask
 
+    def get_line_mask_without_other_objects(self, candidate_list):
+        """
+        Generates a white mask that not contains pixels in the green field, above the field boundary or in the specified candidates.
+
+        :param candidate_list: List ob candidate bounding boxes that are subtracted from the final mask
+        :return: Mask
+        """
+        mask = self.get_line_mask().copy()
```

Comment:
Yes. Because if another step calls `get_line_mask()`  after `get_line_mask_without_other_objects()` it would also return the mask without the other objects. The return value if `get_line_mask()` would become depended whether or not  `get_line_mask_without_other_objects()`  had been called before, because the latter would apply the masking on the same reference.